tests/cli/test_vehicle.py

```diff
@@ -112,6 +118,8 @@
         "GPS Latitude      48.1234567\n"
         "GPS Longitude     11.1234567\n"
         "GPS last updated  2020-02-18 17:58:38\n"
+        "Lock status       locked\n"
+        "Lock last updated 2022-02-02 13:51:13\n"
```

Comment:
You need to add just one space, but on all the lines.
```suggestion
        "Lock status        locked\n"
        "Lock last updated  2022-02-02 13:51:13\n"
```

You will also need to add one dash on the left part of all the corresponding header and footer lines.

Reply:
👍